solr/solr-ref-guide/modules/deployment-guide/pages/configuring-logging.adoc

```diff
@@ -172,6 +172,22 @@ The log file under which you can find all these queries is called `solr_slow_req
 In addition to the logging options described above, it's possible to log only a selected list of request parameters (such as those sent with queries) with an additional request parameter called `logParamsList`.
 See the section on xref:query-guide:common-query-parameters.adoc#logparamslist-parameter[logParamsList Parameter] for more information.
 
+== Selective Logging on SolrCore
+
+It can sometimes be noisy when `SolrCore` logs info on every response. Selective logging can be configured on SolrCore by using the `MarkerFilter` on request paths in `log4j2.xml`.
```

Comment:
[0] Would a reader necessarily know what "SolrCore logs" are?  (i.e. that these are Solr's request logs). Or would it be better to call that out a bit more explicitly?

> Solr logs all core/collection requests at INFO level using the `o.a.s.c.SolrCore.Request` and `o.a.s.c.SolrCore.SlowRequest` loggers.  This can be disabled entirely by changing the level of these loggers to `WARN` or `ERROR`, or administrators can configure more selective request logging on a per-path basis by using `MarkerFilter`...

[Q] Is the marker stuff really only for the request logging in SolrCore, or is it also used for the HttpSolrCall request logging that's used for non-core (i.e. admin) requests?

Reply:
Thank you for the suggestion! I have taken most of ur text and rewrote it, can you please take a look again? 😊 

The marker was only added to SolrCore, as it was the one that produces too much log for us (we have hundreds of cores per node and replication can create be noisy for us)

Reply:
LGTM, thanks for adjusting